### PR TITLE
Update depdencies

### PR DESCRIPTION
**Pull request by Doikor**
Updated finagle, cats, cats-effect, argonaut, fs2, refined, scalatest and discipline

### CODE REVIEW

**Comment by Doikor** [comment]
Styles are broken but @sergeykolbasov is bringin in scalafmt anyway and tut docs are broken (does not support latest scala) and should be updated to mdoc (I am working on these for a separate pr)

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,6 @@ local.*
 
 .bloop
 .metals
+.vscode
 project/metals.sbt
+project/project
```
Comment (kingslef) [inline .gitignore:16]: nitpick but newline?

File: build.sbt [hunk 2]
```diff
@@ -3,21 +3,21 @@ import microsites.ExtraMdFileConfig
...
-lazy val twitterVersion = "20.7.0"
+lazy val twitterVersion = "20.9.0"
 lazy val circeVersion = "0.13.0"
 lazy val circeIterateeVersion = "0.13.0-M2"
 lazy val circeFs2Version = "0.13.0"
 lazy val shapelessVersion = "2.3.3"
-lazy val catsVersion = "2.1.1"
-lazy val argonautVersion = "6.2.4"
+lazy val catsVersion = "2.2.0"
+lazy val argonautVersion = "6.2.5"
```
Comment (kingslef) [inline build.sbt:16]: not 6.3.1?

File: build.sbt [hunk 3]
```diff
@@ -47,9 +47,9 @@ lazy val scala213CompilerOptions = Seq(
 
 val testDependencies = Seq(
   "org.scalacheck" %% "scalacheck" % "1.14.2",
```
Comment (kingslef) [inline build.sbt:49]: there is also 1.14.3

File: build.sbt [hunk 4]
```diff
@@ -47,9 +47,9 @@ lazy val scala213CompilerOptions = Seq(
...
-  "org.scalatest" %% "scalatest" % "3.1.2",
+  "org.scalatest" %% "scalatest" % "3.1.4",
```
Comment (kingslef) [inline build.sbt:50]: there is also [3.2.0](https://www.scalatest.org/release_notes/3.2.0)

File: build.sbt [hunk 5]
```diff
@@ -47,9 +47,9 @@ lazy val scala213CompilerOptions = Seq(
...
   "org.typelevel" %% "cats-laws" % catsVersion,
-  "org.typelevel" %% "discipline-scalatest" % "1.0.0"
+  "org.typelevel" %% "discipline-scalatest" % "1.0.1"
```
Comment (kingslef) [inline build.sbt:52]: there is [2.0.0](https://github.com/typelevel/discipline-scalatest/releases)

**Review by kingslef — COMMENTED** [review]
looks like there are newer versions to some of the packages.

File: .gitignore (hunk 1 above)
Comment (Doikor) [reply .gitignore]: fixed

File: build.sbt (hunk 2 above)
Comment (Doikor) [reply build.sbt]: fixed

File: build.sbt (hunk 3 above)
Comment (Doikor) [reply build.sbt]: fixed

File: build.sbt (hunk 4 above)
Comment (Doikor) [reply build.sbt]: fixed

File: build.sbt (hunk 5 above)
Comment (Doikor) [reply build.sbt]: fixed

File: test/src/main/scala/io/finch/test/ServiceSuite.scala [hunk 6]
```diff
@@ -3,15 +3,15 @@ package io.finch.test
 import com.twitter.finagle.Service
 import com.twitter.finagle.http.{Request, Response}
 import com.twitter.util.{Await, Duration}
-import org.scalatest.{fixture, Outcome}
+import org.scalatest.{FixtureTestSuite, Outcome}
 
 /**
  * A convenience class that is designed to make it easier to test HTTP services
  * both directly and in integration tests that are served locally. Implementing
  * classes must extend [[org.scalatest.fixture.Suite]] through [[org.scalatest.fixture.FlatSpec]]
```
Comment (kingslef) [inline test/src/main/scala/io/finch/test/ServiceSuite.scala:11]: another nitpick but these don't exist anymore.

```suggestion
 * classes must extend [[org.scalatest.FixtureTestSuite]] through [[org.scalatest.flatspec.FixtureAnyFlatSpec]]
```

**Review by kingslef — COMMENTED** [review]
one nitpick but looks good to me!

**Pull request merged**